README.md

```diff
@@ -324,8 +324,8 @@ You can use `--config` option to provide configuration file for the tool. You ca
 
 ## Using CVE Binary Tool in GitHub Actions
 
-If you want to integrate cve-bin-tool as a part of your github action pipeline.
-You can checkout our example [github action](https://github.com/intel/cve-bin-tool/blob/main/doc/how_to_guides/cve_scanner_gh_action.yml).
+If you want to integrate cve-bin-tool as a part of your github action pipeline, you can use cve-bin-tool's official GitHub Action. Find more details [here](https://github.com/intel/cve-bin-tool-action/#cve-binary-tool-github-action). The GitHub Action provide reports on the security tab.
```

Comment:
We should probably add that the Security tab is only available to public open source projects.   If they run the action on a private repo, can they still get reports out of the build artifacts?  If that's the case we should also list that as an option somewhere.

Users wishing to scan private repos can follow the instructions below to call cve-bin-tool directly, regardless.

Reply:
updated the doc!